### PR TITLE
fix: add missing script for building wizard

### PR DESCRIPTION
**Pull request by elevatebart**
As mentioned in #15450 we need to deliver the wizard again and probably update the Readme.md for `@cypress/react`.

### CODE REVIEW

File: npm/react/README.md [hunk 1]
```diff
@@ -94,14 +88,6 @@ module.exports = (on, config) => {
 
 See [Recipes](./docs/recipes.md) for more examples.
 
-3. ⚠️ Turn the experimental component support on in your `cypress.json`. You can also specify where component spec files are located. For example, to have them located in `src` folder use:
-
-```json
-{
-  "experimentalComponentTesting": true,
-  "componentFolder": "src"
```
Comment (dmtrKovalenko) [inline npm/react/README.md:102]: It's still required, isn't it?

**Review by dmtrKovalenko — DISMISSED** [review]

File: npm/react/README.md (hunk 1 above)
Comment (elevatebart) [reply npm/react/README.md:102]: It is not required anymore since we do not use preprocessors.
We should update the wizard, I will make a ticket.

Comment (dmtrKovalenko) [reply npm/react/README.md:102]: No `componentFolder` is still required in order to find distinguish component specs and e2e specs.

Comment (JessicaSachs) [reply npm/react/README.md:102]: Dmitriy is right.

Comment (elevatebart) [reply npm/react/README.md:102]: Why isn't it a default? 
I'll fix it

**Comment by elevatebart** [comment]
`ensureNotAnimating` is failing on windows for some reason. That is the error on appveyor.

Cypress throws the following error:

```js
 animation_coords_history_invalid: 'coordsHistory must be at least 2 sets of coords',
```

I give up for tonight.

**Pull request merged**